in_toto/model.go

```diff
@@ -202,6 +202,17 @@ type SupplyChainItem struct {
 	ExpectedProducts  [][]string `json:"expected_products"`
 }
 
+/*
+validateArtifactRule calls UnpackRule to validate that the passed rule conforms
+with any of the available rule formats.
+*/
+func validateArtifactRule(rule []string) error {
+	if _, err := UnpackRule(rule); err != nil {
+		return err
+	}
+	return nil
+}
+
```

Comment:
Should there be an explicit `TestValidateArtifactRule` for this?

Reply:
IMO we're good here. It's only a wrapper for `UnpackRule`, which is extensively tested in `rulelib_test.go`. And it's covered by higher-level validate functions, such as `TestValidateSupplyChainItem`.